article/test/model/LiveBlogCurrentPageTest.scala

```diff
@@ -62,10 +84,12 @@ class LiveBlogCurrentPageTest extends FlatSpec with Matchers {
   }
 
   it should "add the most recent pinned post to the first page" in {
-    val blocks = fakeBlocks(5, 0, 2)
-    val latestPinnedBlock = fakeBlock(2, false, true).copy(id = "2-pinned")
-    val olderPinnedBlock = fakeBlock(1, false, true)
-    val evenOlderPinnedBlock = fakeBlock(0, false, true)
+    // 5, 4 regular and
```

Comment:
Could we either remove these comments or make them clearer please? Think as they are it's a bit vague :)